src/vs/platform/terminal/common/xterm/shellIntegrationAddon.ts

```diff
@@ -369,7 +388,7 @@ export class ShellIntegrationAddon extends Disposable implements IShellIntegrati
 		const [command] = data.split(';');
 		switch (command) {
 			case ITermOscPt.SetMark: {
-				this._createOrGetCommandDetection(this._terminal).handleGenericCommand({ genericMarkProperties: { disableCommandStorage: true } });
+				this._terminal.registerMarker();
```

Comment:
I think this just creates an `IMarker` and does nothing with it. It should use `_createOrGetBufferMarkDetection` without an id